projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_splitter.scss

```diff
@@ -56,9 +56,34 @@ $_fluent-splitter: extend($_light-splitter);
 /// @requires $_bootstrap-shape-splitter
 $_bootstrap-splitter: extend(
     $_light-splitter,
-    $_bootstrap-shape-splitter,
```

Comment:
Why are you removing the bootstrap-shape-splitter both in here and in the shapes? If it's necessary, remove that it requires it.